### PR TITLE
[Core] Adding ClearNonHistoricalValues to VariableUtils

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

File: kratos/utilities/variable_utils.h [hunk 1]
```diff
@@ -384,11 +384,33 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
         KRATOS_CATCH("")
     }
 
+
+    /**
+     * @brief Clears the container data value container
+     * @param rContainer Reference to the objective container
+     */
+    template< class TContainerType>
+    void ClearNonHistoricalValues(TContainerType& rContainer)
```
Comment (philbucher) [inline kratos/utilities/variable_utils.h]: could you add the same for historical vars (just for nodes ofc) for completeness?

Comment (loumalouomega) [reply kratos/utilities/variable_utils.h]: I don't know, I am not sure if it is possible to clear the database

Comment (loumalouomega) [reply kratos/utilities/variable_utils.h]: I will check, but I prefer to let that for a future PR

Comment (philbucher) [reply kratos/utilities/variable_utils.h]: ok

Another thing that is a bit ambiguous imo is that I though that this function sets all the non-hist vars to zero, but it does sth else.

How abt `ClearNonHistoricalData`? then it does exactly what the name indicates

Comment (loumalouomega) [reply kratos/utilities/variable_utils.h]: That's true

**Comment by loumalouomega** [comment]
Done

File: kratos/utilities/variable_utils.h [hunk 2]
```diff
@@ -384,11 +384,33 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
...
+    void ClearNonHistoricalData(TContainerType& rContainer)
+    {
+        KRATOS_TRY
+
+        const auto it_cont_begin = rContainer.begin();
+
+        #pragma omp parallel for
+        for (int k = 0; k< static_cast<int> (rContainer.size()); ++k) {
+            auto it_cont = it_cont_begin + k;
+            auto& r_data = it_cont->Data();
```
Comment (philbucher) [inline kratos/utilities/variable_utils.h]: I don't think you need this reference, do you?
```suggestion
            it_cont->Data().Clear();
```

Comment (loumalouomega) [reply kratos/utilities/variable_utils.h]: If I apply this suggestion the next line will stay the sanme

Comment (philbucher) [reply kratos/utilities/variable_utils.h]: I know :)

**Review by philbucher — APPROVED** [review]

**Pull request merged**